pkg/scanners/cloudformation/parser/parser.go

```diff
@@ -151,8 +189,48 @@ func (p *Parser) ParseFile(ctx context.Context, fs fs.FS, path string) (context
 	}
 
 	for name, r := range context.Resources {
-		r.ConfigureResource(name, fs, path, context)
+		r.ConfigureResource(name, fsys, path, context)
 	}
 
 	return context, nil
 }
+
+func (p *Parser) parseParams() error {
+	if p.overridedParameters != nil { // parameters have already been parsed
+		return nil
+	}
```

Comment:
when would this case happen? just curious.

Reply:
When parsing the file system, the parameters will be parsed once, not before parsing each CF template.